indico/modules/events/contributions/templates/display/_contribution_list.html

```diff
@@ -0,0 +1,70 @@
+{% macro render_displayed_entries_fragment(displayed_num, total_num) -%}
+    <span id="filtering-state" class="i-button label icon-list"
+          title="{% trans %}Displaying {{displayed_num}} out of {{total_num}} contributions{% endtrans %}">
+        <strong>{{ displayed_num }}</strong> / {{ total_num }}
+    </span>
+{%- endmacro %}
+
+{% macro render_contribution_list(event, contributions) %}
+    <table class="i-table-widget">
+        <thead>
+            <tr>
+                <th>{% trans %}Title{% endtrans %}</th>
+                <th>{% trans %}Start date{% endtrans %}</th>
+                <th>{% trans %}Session{% endtrans %}</th>
+                <th>{% trans %}Track{% endtrans %}</th>
+                <th>{% trans %}Presenters{% endtrans %}</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for contrib in contributions -%}
+                <tr data-friendly-id="{{ contrib.friendly_id }}">
+                    <td data-searchable="{{ contrib.title }}">
+                        <a href="{{ url_for('.display_contribution', contrib) }}">{{ contrib.title }}</a>
+                    </td>
+                    <td>
+                        {% if contrib.timetable_entry -%}
+                            {{ contrib.timetable_entry.start_dt|format_date }}
+                        {% else %}
+                            {% trans %}The contribution has not been scheduled yet.{% endtrans %}
+                        {% endif %}
+                    </td>
+                    <td>
+                        {% if contrib.session %}
+                            <a href="{{ url_for('sessions.display_session', contrib.session) }}">{{ contrib.session.title }}</a>
+                        {% endif %}
+                    <td>{{ contrib.track.title if contrib.track }}</td>
+                    <td>
+                        {% if contrib.speakers -%}
+                            {% set speakers = contrib.speakers|map(attribute='full_name')|join(', ') %}
+                            {% trans speakers=speakers -%}
```

Comment:
`speakers=speakers` is not needed. plain vars can be used without specifying them here